wolnut/main.py

```diff
@@ -7,6 +7,8 @@
 
 logger = logging.getLogger("wolnut")
 
+def battery_percent(ups_status):
+    return int(float(ups_status.get("battery.charge", 100)))
```

Comment:
Should probably use round

```suggestion
    return round(float(ups_status.get("battery.charge", 100)), 0)
```

So number get properly rounded up or down (99.70 -> 100, 99.4 -> 99)

Reply:
Done.

It's not clear to me that rounding up is necessarily preferable to truncating here, but either way doesn't seem likely to matter.